src/SetupRemote/Product.wxs

```diff
@@ -26,6 +26,12 @@
     <DirectoryRef Id="INSTALLFOLDER">
       <Component Id="ProductAssemblies" Guid="{9C717F8F-4A97-4FA5-8031-5A180467754D}" Win64="yes">
         <?include "$(var.SourceDir)\Setup\CommonAssemblies.wxi" ?>
+        <!-- For remote we must include C++ runtime as VS may not be installed -->
+        <?ifdef Debug?>
+        <File Source="$(var.LibDir)\vcruntime140d.dll" />
```

Comment:
This is the debug version of the VC++ runtime. We should only be including this one for debug builds, and the release one for release builds.

Unless we're using a debug version of some library (zlib?) in our release builds. In which case we should switch to release version of that, since there's likely a significant perf difference.